bootstrap/kubeadm/main.go

```diff
@@ -218,9 +220,12 @@ func main() {
 		LeaderElectionResourceLock: resourcelock.LeasesResourceLock,
 		HealthProbeBindAddress:     healthAddr,
 		PprofBindAddress:           profilerAddress,
+		Metrics: metricsserver.Options{
+			BindAddress: metricsBindAddr,
```

Comment:
q: as a follow-up, what about opening an issue for documenting how to get metrics in CAPI?

Reply:
In this PR nothing changes.

Authorized metrics will be part of this effort: https://github.com/kubernetes-sigs/cluster-api/pull/9264
I was planning to write a corresponding issue and bring it up in the office hours.

But just the CR bump itself in this PR doesn't change anything